test/unit/boolean-phrase-parser.spec.js

```diff
@@ -0,0 +1,205 @@
+describe.only('booleanPhraseParser', function () {
```

Comment:
Fixed the name of the describe to be with the name of the class (capital letter at the start)